packages/app/src/components/table/index.tsx

```diff
@@ -237,3 +260,127 @@ const Td: React.FC<BaseProps & { isSticky?: boolean }> = ({
     </td>
   );
 };
+
+const RowObject: React.FC<
+  BaseProps & {
+    rowObject: Data;
+    columns: TableColumn[];
+    close: () => void;
+  }
+> = ({ on, rowObject, columns, close }) => {
+  return (
+    <div className="px-10 py-10 flex flex-col h-full w-full">
+      <div>
+        <Button
+          className="flex mr-0 ml-auto"
+          variant="text"
+          on={COLOR_SYSTEM.SURFACE}
+          Icon={CloseIcon}
+          onClick={close}
+        />
+      </div>
+      <div className="grow overflow-scroll space-y-10">
+        {Object.keys(rowObject).map((objectKey, index) => (
+          <Accordion
+            key={index}
+            on={on}
+            schema={columns.find((column) => column.key === objectKey)?.schema}
+            selectedRowData={rowObject}
+            objectKey={objectKey}
+          />
+        ))}
+      </div>
+    </div>
+  );
+};
+
+const Accordion: React.FC<
```

Comment:
Can this component be used commonly? If so, move it /components/Accordion.

Reply:
This component can be used commonly so move it the right place

Reply:
This accordion only accept object, so can not say it is universal. I will move accordion component `/components/table/`